test/tools/unified-spec-runner/entities.ts

```diff
@@ -169,6 +208,7 @@ export class UnifiedMongoClient extends MongoClient {
     this.observedSdamEvents = (description.observeEvents ?? [])
       .map(e => UnifiedMongoClient.SDAM_EVENT_NAME_LOOKUP[e])
       .filter(e => !!e);
+    this.observedLogEvents = new Map();
```

Comment:
Will initialize this with the observedLogEvents passed in to the constructor in future, but need to implement NODE-4849 to have them take effect